addons/stock/models/stock_move.py

```diff
@@ -1155,6 +1160,29 @@ def unlink(self):
         self.mapped('move_line_ids').unlink()
         return super(StockMove, self).unlink()
 
+    def split_mts(self):
+        total_org = sum(qty for qty in self.move_dest_ids.mapped('move_orig_ids.product_uom_qty'))
+        total_dest = sum(qty for qty in self.move_dest_ids.mapped('product_uom_qty'))
```

Comment:
don't need for loop
total_dest = sum(self.move_dest_ids.mapped('product_uom_qty')